src/modules/ngx-daterange/src/components/datepicker/date-range-picker.component.html

```diff
@@ -8,6 +8,85 @@
       <input [attr.id]="instanceId" class="form-control" type="text" [formControlName]="controlName">
     </ng-container>
 
+    <div class="modal" tabindex="-1" role="dialog" [ngStyle]="{'display':displayStyle}">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+          <div class="modal-body">
+            <div [ngClass]="{'dateRangePickerModal': true,  'dateRangePicker-single': options.singleCalendar, 'is-chrome': checkChrome(), 'mobile': isMobile }">
+              <div class="container-fluid">
+                <div class="row">
+                  <div class="col">
+        
+                    <!-- Manual Inputs -->
+                    <ng-container *ngIf="!isMobile">
```

Comment:
@FinalButterfly This template needs to be DRYed up. With the addition of the modal templating, there are now several sections that need so be abstracted into separate components.

Reply:
Also, the modal backdrop is missing when opening the datepicker as a modal.

Reply:
I thought I would keep it in the same file for simplicity, but ok, no problem. Of course, I'll add the backdrop too.